RadixWallet/Features/AssetsFeature/Components/NonFungibleAssetList/Components/Row/NonFungbileAssetRow+Reducer.swift

```diff
@@ -146,3 +149,13 @@ extension NonFungibleAssetList {
 		}
 	}
 }
+
+private extension NonFungibleAssetList.Row.InternalAction.TokensLoadResult {
+	init(tokens: [OnLedgerEntity.NonFungibleToken], hiddenAssets: [AssetAddress], nextPageCursor: String?, previousTokenIndex: Int) {
```

Comment:
split lines please (is it CTRL + M ?) :)

Reply:
TIL about that shortcut, thanks!